pkg/kubelet/kubelet.go

```diff
@@ -225,6 +227,8 @@ type Kubelet struct {
 	// how long to keep idle streaming command execution/port forwarding
 	// connections open before terminating them
 	streamingConnectionIdleTimeout time.Duration
+
+	eventSource api.EventSource
```

Comment:
Yeah, here too-- if you make this a record.EventRecorder it's also easier for testing.